### PR TITLE
Update build, close streams in case of exception

### PR DESCRIPTION
**Pull request by elevenfive**
- Add support for Glide 3.7.0
- Update to Android Gradle Plugin 2.1.0
- Use Support Library 23.4.0
- Use Compile SDK 23
- Use Build Tools 23.0.3
- Use Gradle 2.13

### CODE REVIEW

File: scissors/src/main/java/com/lyft/android/scissors/Utils.java [hunk 1]
```diff
@@ -64,17 +65,25 @@ public static Bitmap asBitmap(Drawable drawable, int minWidth, int minHeight) {
...
                         Log.e(TAG, "Error attempting to save bitmap.", throwable);
                     }
+                } finally {
+                    try {
+                        if (outputStream != null) {
+                            outputStream.close();
+                        }
+                    } catch (Exception e) {
+                        Log.e(TAG, "Error attempting to save bitmap.", e);
+                    }
```
Comment (eveliotc) [inline scissors/src/main/java/com/lyft/android/scissors/Utils.java]: Please create `closeQuietly` method

File: scissors/src/main/java/com/lyft/android/scissors/Utils.java [hunk 2]
```diff
@@ -92,13 +101,18 @@ public void run() {
...
                     if (BuildConfig.DEBUG) {
                         Log.e(TAG, "Error attempting to save bitmap.", throwable);
                     }
+                } finally {
+                    if (closeWhenDone) {
+                        try {
+                            outputStream.close();
+                        } catch (Exception e) {
+                            Log.e(TAG, "Error attempting to save bitmap.", e);
+                        }
```
Comment (eveliotc) [inline scissors/src/main/java/com/lyft/android/scissors/Utils.java]: Please use `closeQuietly` method here as well

File: scissors/src/main/java/com/lyft/android/scissors/Utils.java (hunk 1 above)
Comment (elevenfive) [inline scissors/src/main/java/com/lyft/android/scissors/Utils.java]: done, ty for feedback!

**Comment by eveliotc** [comment]
@elevenfive thanks for this, as per contributing guidelines [can you please sign Lyft's CLA?](https://oss.lyft.com/cla)

**Comment by elevenfive** [comment]
Done

**Comment by eveliotc** [comment]
Awesome thanks!

**Pull request merged**